realm/src/main/java/io/realm/Realm.java

```diff
@@ -1588,6 +1155,22 @@ static String getCanonicalPath(File realmFile) {
         }
     }
 
+    // Return all handlers registered for this Realm
```

Comment:
Method is package protected so it is internal only. Do you want me to remove it or make it a proper JavaDoc comment?